tests/test_config.py

```diff
@@ -25,6 +26,9 @@
 from mlrun.db.httpdb import HTTPRunDB
 
 ns_env_key = f"{mlconf.env_prefix}NAMESPACE"
+dfpr_env_key = f"{mlconf.env_prefix}DEFAULT_FUNCTION_POD_RESOURCES__"
```

Comment:
Remember that most of the time we're reading code not writing code, therefore we try to optimize readability (and not the amount of characters we're typing)
Imagine someone who is reading line 30 or 31, they would have no clue what `dfpr_env_key` holds, simply cause `dfpr` means nothing, you require them to go the definition of it to understand what it is, i.e. hard to read code.
Therefore, I don't see a problem with naming it `default_function_pod_resources_env_key`
I obviously understand you just followed the "file convention" going after that `ns_env_key` above, breaking convention may definitely hurt readability, here I don't feel like it is, also `ns` do make some sense, for example you can do `kubectl get ns` (instead of `get namespaces`)
Of course you can just change it as well